server/v2/cometbft/utils.go

```diff
@@ -268,6 +270,35 @@ func QueryResult(err error, debug bool) *abci.QueryResponse {
 	}
 }
 
+func gRPCErrorToSDKError(err error) *abci.QueryResponse {
+	toQueryResp := func(sdkErr *errorsmod.Error, err error) *abci.QueryResponse {
+		return &abci.QueryResponse{
+			Code:      sdkErr.ABCICode(),
+			Codespace: sdkErr.Codespace(),
+			Log:       err.Error(),
+		}
```

Comment:
might no need the `rpc error` prefix but just error msg
```suggestion
		res := &abci.QueryResponse{
			Code:      sdkErr.ABCICode(),
			Codespace: sdkErr.Codespace(),
		}
		type grpcstatus interface{ GRPCStatus() *status.Status }
		if grpcErr, ok := err.(grpcstatus); ok {
			res.Log = grpcErr.GRPCStatus().Message()
		} else {
			res.Log = err.Error()
		}
		return res
```

Reply:
Good one, thanks!